contracts/FlowIDTableStaking_new.cdc

```diff
@@ -1275,6 +1350,21 @@ pub contract FlowIDTableStaking {
             FlowIDTableStaking.account.save<Int>(newLimit, to: /storage/idTableCandidateNodeLimit)
         }
 
+        /// Set slot (count) limits for each node type (role).
+        pub fun setSlotLimits(slotLimits: {UInt8: UInt16}) {
```

Comment:
🏁  There isn't a default value for this in the upgrade path. We will need to manually set this value while upgrading the contract.

Reply:
yep, that'll be part of my upgrade plan once this is all finalized